crates/torii/src/cli.rs

```diff
@@ -78,13 +98,29 @@ async fn main() -> anyhow::Result<()> {
 
     let provider = JsonRpcClient::new(HttpTransport::new(Url::parse(&args.rpc).unwrap()));
 
-    let manifest = if let Some(manifest_path) = args.manifest {
-        Manifest::load_from_path(manifest_path).expect("Failed to load manifest")
+    // Load manifest
+    let manifest_path = scarb::ops::find_manifest_path(None)?;
+    let config = Config::builder(manifest_path.clone())
+        .log_filter_directive(env::var_os("SCARB_LOG"))
+        .build()?;
+    let ws = scarb::ops::read_workspace(config.manifest_path(), &config)?;
+    let target_dir = ws.target_dir().path_existent().unwrap();
+    let target_dir = target_dir.join(ws.config().profile().as_str());
+    let manifest = Manifest::load_from_path(target_dir.join("manifest.json"))
+        .expect("Failed to load manifest");
+
+    // Get world address
```

Comment:
First check world_address is passed at argument
Second if not, get address from manifest
Third if not, get address from dojo env metadata from scarb.